### PR TITLE
Add Gleam App by @akarataev (#690)

### PR DESCRIPTION
**Pull request by akarataev**
1. [x] Project URL:
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github-username
7. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -17275,6 +17275,26 @@
...
+        "health",
+        "viper"
+      ],
+      "tags": [
+        "swift",
+        "coreml",
+        "avkit",
+        "viper"
+      ],
+      "description": "Gleam is mobile application (based on machine learning algorithm) that will help make an early diagnosis of melanoma and get the necessary treatment on time.",
```
Comment (vpeschenkov) [inline contents.json]: @akarataev could you remove the period (.) at the end of description?

**Comment by vpeschenkov** [comment]
Hello @akarataev, thank you for the contribution. Could you resolve the review issue?

File: contents.json (hunk 1 above)
Comment (akarataev) [reply contents.json]: @vpeschenkov well, I updated my commit.

**Comment by akarataev** [comment]
Hello @vpeschenkov. Yes, I fixed this issue.

**Comment by vpeschenkov** [comment]
@akarataev thank you!

**Comment by sriprat5445** [comment]
😅😂🤣

**Pull request merged**